app/models/authorization.rb

```diff
@@ -12,7 +18,7 @@ class Authorization
 
   embedded_in :user
 
-  scope :external_providers, -> { where('provider.ne': 'tomatoes') }
+  scope :external_providers, -> { nin(provider: INTERNAL_PROVIDERS) }
```

Comment:
Why not the opposite, that would also be the *non-negated* case? I mean why defining the external providers scope in terms of `INTERNAL_PROVIDERS`?